### PR TITLE
MCKIN-17533 korean Translations added

### PR DESCRIPTION
**Pull request by musmanmalik**
[https://edx-wiki.atlassian.net/browse/MCKIN-17533](https://edx-wiki.atlassian.net/browse/MCKIN-17533)

### CODE REVIEW

**Comment by musmanmalik** [comment]
@xitij2000 This one related to korean translations as informed you earlier by team. Please review this. Thanks

**Comment by xitij2000** [comment]
@musmanmalik  Note that problem builder 4 is incompatible with solutions. Please also make a PR against the problem-builder-v3 branch.

**Comment by musmanmalik** [comment]
@xitij2000  here is the pr for v3 https://github.com/open-craft/problem-builder/pull/284 kindly review both.

File: problem_builder/public/js/translations/ko_KR/textjs.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+}(this));
+
+
+                  }
+                };
+                ProblemBuilderXBlockI18N.init();
+                global.ProblemBuilderXBlockI18N = ProblemBuilderXBlockI18N;
+            }(this));
+
```
[the remark this answers is not in the document]
Comment (xitij2000) [reply problem_builder/public/js/translations/ko_KR/textjs.js:165]: @taniwha This file is auto-generated, as such its formatting should be ignored. If this is the only issue it's OK to merge this.

[the remark this answers is not in the document]
Comment (musmanmalik) [reply problem_builder/public/js/translations/ko_KR/textjs.js:165]: Thanks @xitij2000 for replying.

[the remark this answers is not in the document]
Comment (taniwha) [reply problem_builder/public/js/translations/ko_KR/textjs.js:165]: ah, if auto-generated, then it can't be helped

**Pull request merged**